frontend/src/scenes/insights/InsightTabs/FunnelTab/FunnelTab.tsx

```diff
@@ -30,7 +30,7 @@ import { TaxonomicFilterGroupType } from 'lib/components/TaxonomicFilter/types'
 export function FunnelTab(): JSX.Element {
     const { insightProps } = useValues(insightLogic)
     const { loadResults } = useActions(insightLogic)
-    const { isStepsEmpty, filters, clickhouseFeaturesEnabled, aggregationTargetLabel } = useValues(
+    const { isStepsEmpty, filters, clickhouseFeaturesEnabled, aggregationTargetLabel, steps } = useValues(
```

Comment:
I think `selectors.steps` is a selector that's dependent on the results we get after hitting the api. In the case where results are empty but filters.length === 1, the delete button will still show. I think what we want here is `selectors.filterSteps` or something similar

Reply:
Good catch, thanks! Just updated.